twine/commands/upload.py

```diff
@@ -25,6 +26,8 @@
 from twine import settings
 from twine import utils
 
+logger = logging.getLogger("twine")
```

Comment:
I think we should follow the de-facto `logger` pattern, which will get a logger for `twine.commands.upload`, which will inherit the configuration of the `twine` logger.

```suggestion
logger = logging.getLogger(__name__)
```

The same applies to `twine.utils`.